.github/workflows/integration.yml

```diff
@@ -54,7 +54,7 @@ jobs:
               restore-keys: ${{ runner.os }}-composer-
 
           - name: Composer Install
-            run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
+            run: composer install --ansi --prefer-dist --no-interaction --no-progress
```

Comment:
This is temporary right?

Reply:
No, we want to keep information when running under CI context